tests/test_vector_sqlite.py

```
@@ -162,39 +160,37 @@ def test_vectorstore_setup_wrong_db_type(self):
         )()
 
         with patch(
-            "src.synapso_core.persistence.implementations.vector_store.vector_sqlite.get_config",
+            "synapso_core.data_store.backend.sqlite.sqlite_vector_store.get_config",
             return_value=mock_config,
         ):
             with pytest.raises(ValueError, match="Vector store type is not sqlite"):
-                with VectorSqliteAdapter() as adapter:
-                    adapter.vectorstore_setup()
+                adapter = SqliteVectorStore()
```

Ruff (0.12.2)

Local variable `adapter` is assigned to but never used

Remove assignment to unused variable `adapter`

(F841)